classes/03class/exercises/c03-make01/fabioacp/Makefile

```diff
@@ -0,0 +1,24 @@
+CREATED_BY ?= Fabio Pereira
+DOCKER_HUB ?= facp
+
+SHORT_COMMIT_SHA ?= $(shell git rev-parse --short HEAD)
+
+IMAGE_NAME ?= make01
+PORT ?= 8081
+
+
+.PHONY: build
+build:
+	@echo "This image was created by '$(CREATED_BY)', based on the '${SHORT_COMMIT_SHA}'" >> index.html
+	docker build -t ${IMAGE_NAME}:${SHORT_COMMIT_SHA} .
+	@rm index.html
+
+.PHONY: run
+run:
+	docker run -dp ${PORT}:80 $(IMAGE_NAME):$(SHORT_COMMIT_SHA)
```

Comment:
what's the difference between a variable between `${}` and `$()`?

Reply:
As far as I know, there is no difference. Is there any?

Reply:
nope.. no difference.. just wandering why you had both in your code. :)